rtrlib/transport/ssh/ssh_transport.c

```diff
@@ -137,14 +137,29 @@ int tr_ssh_open(void *socket)
 		goto error;
 	}
 
+	if (config->client_privkey_path) {
+		SSH_DBG("%s: Trying publickey authentication", ssh_socket, __func__);
+		ssh_options_set(ssh_socket->session, SSH_OPTIONS_IDENTITY, config->client_privkey_path);
+
+		int rtval;
+
```

Comment:
is it possible to swap the order:
'int rtval'
before ssh_options_set, 

like most of C programs ?

Reply:
I prefer to declare variables were they are first used not at the beginning at the block. The declaration is only "freestanding" because it wanted to avoid duplication the declaration in the following if/else macro.